### PR TITLE
Update tables.js to add top for datatable DOM

### PR DESCRIPTION
**Pull request by crochefort**
More information in #9709

When you are adding the option DOM of Datatable, to add the pagination at the top of the table, when selecting more item than what the table is containing, 60 item vs 100 to show, the pagination stay at the top!

This just remove the pagination at the top like at the bottom.

Thanks to @wewhite

### CODE REVIEW

**Review by duboisp — COMMENTED** [review]
Pre-approve once: 
* local testing is completed
* The new empty line is added before the comment at line 181

File: src/plugins/tables/tables-en.hbs [hunk 1]
```diff
@@ -970,6 +970,127 @@
 			</tr>
 		</tbody>
 	</table>
+
+	<p>Adding the pagination at the top of the table and keeping it at the bottom</p>
```
Comment (duboisp) [inline src/plugins/tables/tables-en.hbs:974]: Please add a subheading to have a clearer separation between both working example.

```suggestion
	<h4>With top pagination example</h4>
	<p>Adding the pagination at the top of the table and keeping it at the bottom by using the following configuration: <code>data-wb-tables='{"ordering" : false, "dom":"&lt;\u0027top\u0027pfil>rt&lt;\u0027bottom\u0027p>&lt;\u0027clear\u0027>"}'</code></p>
```

File: src/plugins/tables/tables-fr.hbs [hunk 2]
```diff
@@ -978,6 +978,131 @@
 			</tr>
 		</tbody>
 	</table>
+
+	<p>Ajouter la pagination au-dessus du tableau en le gardant en dessous.</p>
```
Comment (duboisp) [inline src/plugins/tables/tables-fr.hbs:982]: Please add a subheading to have a clearer separation between both working example.

```suggestion
	<h4>Exemple avec une pagination en haut du tableau</h4>
	<p>Ajouter la pagination au-dessus du tableau en le gardant en dessous en utilisant la configuration suivante: <code>data-wb-tables='{"ordering" : false, "dom":"&lt;\u0027top\u0027pfil>rt&lt;\u0027bottom\u0027p>&lt;\u0027clear\u0027>"}'</code></p>
```

File: src/plugins/tables/tables.js [hunk 3]
```diff
@@ -172,11 +173,12 @@ $document.on( "draw.dt", selector, function( event, settings ) {
...
 	) {
 		pagination.addClass( "hidden" );
+		pagination_top.addClass( "hidden" );
 	} else {
 
 		// Make sure Pagination is visible
 		pagination.removeClass( "hidden" );
-
+		pagination_top.removeClass( "hidden" );
 		// Update Pagination List
```
Comment (duboisp) [inline src/plugins/tables/tables.js:182]: Besoin d'une espace qui précède tout commentaire

```suggestion
		pagination_top.removeClass( "hidden" );

		// Update Pagination List
```

File: src/plugins/tables/tables.js
Comment (duboisp) [inline src/plugins/tables/tables.js:1]: The top pagination, if it do exist, will need to be structured the same as the bottom pagination which leverage an ordered list as per the logic written for the bottom pagination from line 183 to 214.

**Review by duboisp — DISMISSED** [review]

File: src/plugins/tables/tables.js
Comment (duboisp) [reply src/plugins/tables/tables.js:1]: I made the change to render the top pagination semantically the same as the bottom pagination.

**Comment by duboisp** [comment]
@Garneauma Since that I added a commit which do impact this change request, can you test locally and review this PR. Thank you

**Review by Garneauma — APPROVED** [review]

**Pull request merged**